test/honeycomb-otel-sdk.test.ts

```diff
@@ -8,20 +8,40 @@ test('it should extend the OTel WebSDK', () => {
   expect(honeycomb).toBeInstanceOf(WebSDK);
 });
 
-/* This test relies on `getResourceAttributes`, a method not
+/* These test relies on `getResourceAttributes`, a method not
```

Comment:
if we're adjusting anyway...
```suggestion
/* These tests rely on `getResourceAttributes`, a method not
```